src/models/individual/usePostIndividual.js

```diff
@@ -22,11 +22,7 @@ export default function usePostIndividual() {
         },
       });
       const successful = get(response, 'status') === 200;
-      const newIndividualGuid = get(response, [
-        'data',
-        'result',
-        'id',
-      ]);
+      const newIndividualGuid = get(response, ['data', 'guid']);
```

Comment:
Just want to confirm that the 'result' is no longer needed here?

Reply:
I think this changed in https://github.com/WildMeOrg/houston/pull/552.

Reply:
Correct